broker/persistence/sqlite/plugin.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2019 UFCG-LSD.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from broker.persistence.persistence_interface import PersistenceInterface
+from broker.persistence.sqlite.model import KubeJobsState
+
+import dill
+import peewee
+
+
+class SqlitePersistence(PersistenceInterface):
+
+    def __init__(self):
+        try:
+            KubeJobsState.create_table()
```

Comment:
this implementation seems to be highly tied to KubeJobs plugin. shouldn't it be more generic and plugin-agnostic?